### ISSUE TITLE
Cannot find include file when using a variable

### ISSUE THREAD

**Issue by omus** [tonyhffong/Lint.jl#123]
When Lint.jl checks for that an included file exists it appears does handle variables correctly:
```julia
julia> using Lint

julia> open("test.jl", "w") do f
           write(f, "println(\"helloworld!\")")
       end
22

julia> script = "test.jl"
"test.jl"

julia> include(script)
helloworld!

julia> lintstr("script = \"test.jl\"; include(script)")
1-element Array{Lint.LintMessage,1}:
 none:1 [               ] FATAL  cannot find include file: ./script
```

**Comment by tonyhffong** [comment]
What should be the default behavior? Programmatically assembled input string for ```include()``` is always going to be iffy, because either Lint ignore it completely or it would try to evaluate the code, which could have un-controlled run time cost and other nastiness.

**Comment by tonyhffong** [comment]
Perhaps it should just get a warning that says `include with a non-literal input cannot be linted properly`. I'm okay with that.

**Comment by TotalVerb** [comment]
I feel like that warning is not needed for non-literal inputs. Almost all problems with `include` are caught very early at runtime, so a static checker shouldn't bother.

**Comment by omus** [comment]
Seems like the best solution is to avoid warning users about dynamic includes.

**Comment by tonyhffong** [comment]
I struggle a bit with whether it should be at least an "informational". Afterall, skipping an include may very well lead to false positives down the line, as we may skip over crucial globals, functions and types definitions. An informational at least gives the user a fighting chance to discover the origins of the false positives.

**Comment by omus** [comment]
I am alright if dynamic includes give an "informational" message. I think that the original warning should be restricted to includes only using literals.

**Comment by Michael-Klassen** [comment]
Should we make the new message an `Info` or `Warning` level message?

**Comment by Michael-Klassen** [comment]
Any suggestions on the message?
How does `Unable to eval include path` sound?

**Comment by tonyhffong** [comment]
How about `Lint would only follow literal include path` or `Lint does not follow non-literal include path`? I think `Info` is sufficient.

**Comment by omus** [comment]
How about `Unable to verify non-literal include file exists`?

**Comment by Michael-Klassen** [comment]
`Unable to follow non-literal include files`?

**Comment by omus** [comment]
Since include only takes a single file path at a time it should probably just be "file". I think that "exists" is clearer than "follows" but I fully admit I'm not familiar with how the lint messages are typically worded.

**Comment by Michael-Klassen** [comment]
I was thinking "follows" because other warnings might occur because the file contents were not included and it might help the user understand why the other errors are there.

**Comment by Michael-Klassen** [comment]
#124
